chart/testdata/frobnitz/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
This is test data. It's a complete chart that the test suite uses to test whether it parses a standards-compliant chart. I can remove the text... but it's just here because it's test data.